docs/core-manage-asdf-vm.md

```diff
@@ -184,6 +192,36 @@ Add the following to `~/.zshrc`:
 
 ?> Completions will need to be [configured as per Homebrew's instructions](https://docs.brew.sh/Shell-Completion#configuring-completions-in-zsh).
 
+### --Linux,Bash,Pacman--
+
+Add the following to `~/.bashrc`:
+
+```shell
+. /opt/asdf-vm/asdf.sh
+```
+
+?> Completions are automatically configured on installation by the aur package.
```

Comment:
[`bash-completion` needs to be installed for the completions to work.](https://wiki.archlinux.org/index.php/bash#Common_programs_and_options)